### PR TITLE
added cypher keyboard support

### PR DESCRIPTION
**Pull request by westfoxtrot**
## Description

Added support for the cypher PCB 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/westfoxtrot/cypher/cypher.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef CYPHER_H
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/cypher.h]: We can replace this include guard and all others with `#pragma once`

```suggestion
#pragma once
```

File: keyboards/westfoxtrot/cypher/cypher.h [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+#define CYPHER_H
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/cypher.h]:
```suggestion

```

File: keyboards/westfoxtrot/cypher/cypher.h [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+#define LAYOUT_ANSI( \
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/cypher.h]: Our naming convention is actually LAYOUT_all_lower_case_description_here. Could you make these all lower case please. 

```suggestion
#define LAYOUT_ansi( \
```

File: keyboards/westfoxtrot/cypher/cypher.h [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    { K30, ___, K32, K33, K34, K35, K36, K37, K38, K39 }, \
+    { K40, K41, K42, ___, ___, ___, K46, ___, ___, K49 }, \
+    { K50, K51, K52, K53, K54, K55, K56, K57, K58 }, \
+    { K60, K61, K62, K63, ___, K65, K66, K67, K68 }, \
+    { K70, K71, ___, K73, ___, K75, K76, K77, K78 }, \
+    { K80, K81, K82, K83, ___, K85, K86, K87, K88 }, \
+    { K90, K91, K92, K93, ___, K95, K96, K97, K98 } \
+}
+
+#define LAYOUT_ISO( \
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/cypher.h]:
```suggestion
#define LAYOUT_iso( \
```

File: keyboards/westfoxtrot/cypher/info.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "keyboard_name": "Cypher",
+  "width": 19.5,
+  "height": 5,
+  "layouts": {
+    "LAYOUT_ANSI": {
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/info.json]:
```suggestion
    "LAYOUT_ansi": {
```

File: keyboards/westfoxtrot/cypher/info.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    },
+
+    "LAYOUT_ISO": {
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/info.json]:
```suggestion
    "LAYOUT_iso": {
```

File: keyboards/westfoxtrot/cypher/keymaps/default-iso/keymap.c [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/keymaps/default-iso/keymap.c]: This is actually already included when you use `#include QMK_KEYBOARD_H`

```suggestion

```

File: keyboards/westfoxtrot/cypher/keymaps/default-iso/keymap.c [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+#define _MA 0
+#define _F1 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_MA] = LAYOUT_ISO( /* Base */
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/keymaps/default-iso/keymap.c]:
```suggestion
[_MA] = LAYOUT_iso( /* Base */
```

File: keyboards/westfoxtrot/cypher/keymaps/default-iso/keymap.c [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+  KC_GESC, KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_BSPC, KC_DEL, 	 	KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,\
+  KC_TAB,    KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,              	 	KC_P7,   KC_P8,   KC_P9,   KC_PPLS,\
+  KC_CAPS,    KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,	KC_QUOT,KC_NUHS,     KC_ENT,  	 	KC_P4,   KC_P5,   KC_P6,   KC_PEQL,\
+  KC_LSFT,KC_NUBS,KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH,        KC_RSFT,        KC_UP,  	KC_P1,   KC_P2,   KC_P3,   KC_PENT,\
+  KC_LCTRL, KC_LGUI,KC_LALT,                KC_SPC,                        KC_RALT,KC_RGUI,MO(_F1),              KC_LEFT,KC_DOWN,KC_RGHT,    KC_P0,   KC_PDOT, KC_BSPC),
+
+[_F1] = LAYOUT_ISO( /* Function Layer */
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/keymaps/default-iso/keymap.c]:
```suggestion
[_F1] = LAYOUT_iso( /* Function Layer */
```

File: keyboards/westfoxtrot/cypher/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/keymaps/default/keymap.c]:
```suggestion

```

File: keyboards/westfoxtrot/cypher/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+#define _MA 0
+#define _F1 1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[_MA] = LAYOUT_ANSI( /* Base */
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/keymaps/default/keymap.c]:
```suggestion
[_MA] = LAYOUT_ansi( /* Base */
```

File: keyboards/westfoxtrot/cypher/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+  KC_GESC, KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,   KC_0,   KC_MINS, KC_EQL, KC_BSPC, KC_DEL, 	 	KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,\
+  KC_TAB,    KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC,	    KC_BSLS, 	 	KC_P7,   KC_P8,   KC_P9,   KC_PPLS,\
+  KC_CAPS,    KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,	KC_QUOT,             KC_ENT,  	 	KC_P4,   KC_P5,   KC_P6,   KC_PEQL,\
+  KC_LSFT,      KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,KC_DOT,  KC_SLSH,        KC_RSFT,        KC_UP,  	KC_P1,   KC_P2,   KC_P3,   KC_PENT,\
+  KC_LCTRL, KC_LGUI,KC_LALT,                KC_SPC,                        KC_RALT,KC_RGUI,MO(_F1),              KC_LEFT,KC_DOWN,KC_RGHT,    KC_P0,   KC_PDOT, KC_BSPC),
+
+[_F1] = LAYOUT_ANSI( /* Function Layer */
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/keymaps/default/keymap.c]:
```suggestion
[_F1] = LAYOUT_ansi( /* Function Layer */
```

File: keyboards/westfoxtrot/cypher/readme.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+# cypher
+
+![cypher](https://github.com/westfoxtrot/Cypher_PCB)
+
+PCB for the cypher keyboard
+
+Keyboard Maintainer: [westfoxtrot](https://github.com/westfoxtrot)
+Hardware Supported: Cypher PCB
+Hardware Availability: with group buy + Files released here (https://github.com/westfoxtrot/Cypher_PCB)
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/readme.md]: I'm droooling.

File: keyboards/westfoxtrot/cypher/rules.mk [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+#   Atmel DFU loader    4096
+#   LUFA bootloader     4096
+#   USBaspLoader        2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=4096
+
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Virtual DIP switch configuration(+1000)
```
Comment (mechmerlin) [inline keyboards/westfoxtrot/cypher/rules.mk]: Does this board have a physical switch? Is it easily accessible by customers through the bottom of the case? If the answer is both yes, ignore my next statement. 

This should be set to `lite` making the switch located at `K00` the "reset" key.

File: keyboards/westfoxtrot/cypher/readme.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
+# cypher
+
+![cypher](https://github.com/westfoxtrot/Cypher_PCB)
+
+PCB for the cypher keyboard
+
+Keyboard Maintainer: [westfoxtrot](https://github.com/westfoxtrot)
```
Comment (noroadsleft) [inline keyboards/westfoxtrot/cypher/readme.md]: Lines 7 and 8 need to end in two spaces for GitHub to render this file properly.

```suggestion
Keyboard Maintainer: [westfoxtrot](https://github.com/westfoxtrot)  
```

File: keyboards/westfoxtrot/cypher/readme.md [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+Hardware Supported: Cypher PCB
```
Comment (noroadsleft) [inline keyboards/westfoxtrot/cypher/readme.md]:
```suggestion
Hardware Supported: Cypher PCB  
```

File: keyboards/westfoxtrot/cypher/config.h [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+ * the keyboard. They are best used in combination with the HID Listen program,
+ * found here: https://www.pjrc.com/teensy/hid_listen.html
+ *
+ * The options below allow the magic key functionality to be changed. This is
+ * useful if your keyboard/keypad is missing keys and you want magic key support.
+ *
+ */
+
+/* key combination for magic key command */
+#define IS_COMMAND() ( \
```
Comment (drashna) [inline keyboards/westfoxtrot/cypher/config.h]: Could you remove this? 

It's not set by default. So it's only needed if you're not using both shifts for the Command feature.

File: keyboards/westfoxtrot/cypher/cypher.c [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+	// put your per-action keyboard code here
+	// runs for every action, just before processing by the firmware
+
+	return process_record_user(keycode, record);
+}
+
+void led_set_kb(uint8_t usb_led) {
```
Comment (drashna) [inline keyboards/westfoxtrot/cypher/cypher.c]: Could you update this function, so it is more in line with: https://docs.qmk.fm/#/custom_quantum_functions?id=example-led_set_user-implementation

File: keyboards/westfoxtrot/cypher/rules.mk (hunk 14 above)
Comment (drashna) [reply keyboards/westfoxtrot/cypher/rules.mk]: You can also add some defines to your `config.h` to change the specific key that you need to press, but "Escape" is a good key

Comment (westfoxtrot) [reply keyboards/westfoxtrot/cypher/rules.mk]: it has a physical switch underneath the spacebar so technically the top of the case but yes it's easily accessed.

**Comment by westfoxtrot** [comment]
@drashna @noroadsleft @mechmerlin all changes made as per your requests

**Comment by drashna** [comment]
Awesome, thanks!

File: keyboards/westfoxtrot/cypher/info.json [hunk 19]
```diff
@@ -0,0 +1,14 @@
...
...":15.5, "y":2}, {"label":"5", "x":16.5, "y":2}, {"label":"6", "x":17.5, "y":2}, {"label":"=", "x":18.5, "y":2}, {"label":"Shift", "x":0, "y":3, "w":2.25}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":1.75}, {"label":"1", "x":15.5, "y":3}, {"label":"2", "x":16.5, "y":3}, {"label":"3", "x":17.5, "y":3}, {"label":"Enter", "x":18.5, "y":3}, {"label":"Up", "x":14.25, "y":3.25}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4}, {"label":"Alt", "x":2.25, "y":4, "w":1.25}, {"label":"Space", "x":3.5, "y":4, "w":6}, {"label":"Alt", "x":9.5, "y":4, "w":1.25}, {"label":"Win", "x":10.75, "y":4}, {"label":"Ctrl", "x":11.75, "y":4, "w":1.25}, {"label":"0", "x":16.5, "y":4}, {"label":".", "x":17.5, "y":4}, {"label":"Backspace", "x":18.5, "y":4}, {"label":"Left", "x":13.25, "y":4.25}, {"label":"Down", "x":14.25, "y":4.25}, {"label":"Right", "x":15.25, "y":4.25}]
```
Comment (noroadsleft) [inline keyboards/westfoxtrot/cypher/info.json]:
```suggestion
      "layout": [{"label":"~", "x":0, "y":0}, {"label":"!", "x":1, "y":0}, {"label":"@", "x":2, "y":0}, {"label":"#", "x":3, "y":0}, {"label":"$", "x":4, "y":0}, {"label":"%", "x":5, "y":0}, {"label":"^", "x":6, "y":0}, {"label":"&", "x":7, "y":0}, {"label":"*", "x":8, "y":0}, {"label":"(", "x":9, "y":0}, {"label":")", "x":10, "y":0}, {"label":"_", "x":11, "y":0}, {"label":"+", "x":12, "y":0}, {"label":"Backspace", "x":13, "y":0}, {"label":"~", "x":14, "y":0}, {"label":"Num Lock", "x":15.5, "y":0}, {"label":"/", "x":16.5, "y":0}, {"label":"*", "x":17.5, "y":0}, {"label":"-", "x":18.5, "y":0}, {"label":"Tab", "x":0, "y":1, "w":1.5}, {"label":"Q", "x":1.5, "y":1}, {"label":"W", "x":2.5, "y":1}, {"label":"E", "x":3.5, "y":1}, {"label":"R", "x":4.5, "y":1}, {"label":"T", "x":5.5, "y":1}, {"label":"Y", "x":6.5, "y":1}, {"label":"U", "x":7.5, "y":1}, {"label":"I", "x":8.5, "y":1}, {"label":"O", "x":9.5, "y":1}, {"label":"P", "x":10.5, "y":1}, {"label":"{", "x":11.5, "y":1}, {"label":"}", "x":12.5, "y":1}, {"label":"|", "x":13.5, "y":1, "w":1.5}, {"label":"7", "x":15.5, "y":1}, {"label":"8", "x":16.5, "y":1}, {"label":"9", "x":17.5, "y":1}, {"label":"+", "x":18.5, "y":1}, {"label":"Caps Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"\"", "x":11.75, "y":2}, {"label":"Enter", "x":12.75, "y":2, "w":2.25}, {"label":"4", "x":15.5, "y":2}, {"label":"5", "x":16.5, "y":2}, {"label":"6", "x":17.5, "y":2}, {"label":"=", "x":18.5, "y":2}, {"label":"Shift", "x":0, "y":3, "w":2.25}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":1.75}, {"label":"Up", "x":14.25, "y":3.25}, {"label":"1", "x":15.5, "y":3}, {"label":"2", "x":16.5, "y":3}, {"label":"3", "x":17.5, "y":3}, {"label":"Enter", "x":18.5, "y":3}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4}, {"label":"Alt", "x":2.25, "y":4, "w":1.25}, {"label":"Space", "x":3.5, "y":4, "w":6}, {"label":"Alt", "x":9.5, "y":4, "w":1.25}, {"label":"Win", "x":10.75, "y":4}, {"label":"Ctrl", "x":11.75, "y":4, "w":1.25}, {"label":"Left", "x":13.25, "y":4.25}, {"label":"Down", "x":14.25, "y":4.25}, {"label":"Right", "x":15.25, "y":4.25}, {"label":"0", "x":16.5, "y":4}, {"label":".", "x":17.5, "y":4}, {"label":"Backspace", "x":18.5, "y":4}]
```

File: keyboards/westfoxtrot/cypher/info.json [hunk 20]
```diff
@@ -0,0 +1,14 @@
...
...":16.5, "y":2}, {"label":"6", "x":17.5, "y":2}, {"label":"=", "x":18.5, "y":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":1.75}, {"label":"1", "x":15.5, "y":3}, {"label":"2", "x":16.5, "y":3}, {"label":"3", "x":17.5, "y":3}, {"label":"Enter", "x":18.5, "y":3}, {"label":"Up", "x":14.25, "y":3.25}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4}, {"label":"Alt", "x":2.25, "y":4, "w":1.25}, {"label":"Space", "x":3.5, "y":4, "w":6}, {"label":"Alt", "x":9.5, "y":4, "w":1.25}, {"label":"Win", "x":10.75, "y":4}, {"label":"Ctrl", "x":11.75, "y":4, "w":1.25}, {"label":"0", "x":16.5, "y":4}, {"label":".", "x":17.5, "y":4}, {"label":"Backspace", "x":18.5, "y":4}, {"label":"Left", "x":13.25, "y":4.25}, {"label":"Down", "x":14.25, "y":4.25}, {"label":"Right", "x":15.25, "y":4.25}]
```
Comment (noroadsleft) [inline keyboards/westfoxtrot/cypher/info.json]:
```suggestion
      "layout": [{"label":"~", "x":0, "y":0}, {"label":"!", "x":1, "y":0}, {"label":"@", "x":2, "y":0}, {"label":"#", "x":3, "y":0}, {"label":"$", "x":4, "y":0}, {"label":"%", "x":5, "y":0}, {"label":"^", "x":6, "y":0}, {"label":"&", "x":7, "y":0}, {"label":"*", "x":8, "y":0}, {"label":"(", "x":9, "y":0}, {"label":")", "x":10, "y":0}, {"label":"_", "x":11, "y":0}, {"label":"+", "x":12, "y":0}, {"label":"Backspace", "x":13, "y":0}, {"label":"~", "x":14, "y":0}, {"label":"Num Lock", "x":15.5, "y":0}, {"label":"/", "x":16.5, "y":0}, {"label":"*", "x":17.5, "y":0}, {"label":"-", "x":18.5, "y":0}, {"label":"Tab", "x":0, "y":1, "w":1.5}, {"label":"Q", "x":1.5, "y":1}, {"label":"W", "x":2.5, "y":1}, {"label":"E", "x":3.5, "y":1}, {"label":"R", "x":4.5, "y":1}, {"label":"T", "x":5.5, "y":1}, {"label":"Y", "x":6.5, "y":1}, {"label":"U", "x":7.5, "y":1}, {"label":"I", "x":8.5, "y":1}, {"label":"O", "x":9.5, "y":1}, {"label":"P", "x":10.5, "y":1}, {"label":"{", "x":11.5, "y":1}, {"label":"}", "x":12.5, "y":1}, {"label":"Enter", "x":13.75, "y":1, "w":1.25, "h":2}, {"label":"7", "x":15.5, "y":1}, {"label":"8", "x":16.5, "y":1}, {"label":"9", "x":17.5, "y":1}, {"label":"+", "x":18.5, "y":1}, {"label":"Caps Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"\"", "x":11.75, "y":2}, {"label":"~", "x":12.75, "y":2}, {"label":"4", "x":15.5, "y":2}, {"label":"5", "x":16.5, "y":2}, {"label":"6", "x":17.5, "y":2}, {"label":"=", "x":18.5, "y":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":1.75}, {"label":"Up", "x":14.25, "y":3.25}, {"label":"1", "x":15.5, "y":3}, {"label":"2", "x":16.5, "y":3}, {"label":"3", "x":17.5, "y":3}, {"label":"Enter", "x":18.5, "y":3}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4}, {"label":"Alt", "x":2.25, "y":4, "w":1.25}, {"label":"Space", "x":3.5, "y":4, "w":6}, {"label":"Alt", "x":9.5, "y":4, "w":1.25}, {"label":"Win", "x":10.75, "y":4}, {"label":"Ctrl", "x":11.75, "y":4, "w":1.25}, {"label":"Left", "x":13.25, "y":4.25}, {"label":"Down", "x":14.25, "y":4.25}, {"label":"Right", "x":15.25, "y":4.25}, {"label":"0", "x":16.5, "y":4}, {"label":".", "x":17.5, "y":4}, {"label":"Backspace", "x":18.5, "y":4}]
```

Comment (westfoxtrot) [reply keyboards/westfoxtrot/cypher/info.json]: Thanks for pointing this out. I've added the fix:

https://github.com/qmk/qmk_firmware/pull/5481

**Pull request merged**